### DISCUSSION TITLE
Allowlist / whitelist for Trusted Advisor findings

### DISCUSSION THREAD

**Discussion by NMuee** [prowler-cloud/prowler#2876] [Ideas]
Hello Prowler team, 

May I suggest if there is a feature that can allow easy filtering or allowlist for "trustedadvisor_errors_and_warnings" based on the different pillars? or maybe split the findings into five different rules based on pillars? 

Giving some context, there are multiple pillars in trusted advisor such as cost, performance, service limits. 
Personally think that security pillars is more related, while the rest we will not want to track under security compliance
However, with the current set up it is very difficult for us to whitelist them. May we have a feature for the above use case? 

Thank you

**Comment by MrCloudSec** [comment]
Hi @NMuee, that would be a great feature but you can allowlist findings based on the Trusted Advisor Check IDs
